src/data/list/count.lean

```diff
@@ -161,6 +161,16 @@ begin
     { simpa [h, hf.ne h] using IH _ } }
 end
 
+lemma count_le_count_map [decidable_eq β] (l : list α) (f : α → β) (x : α):
```

Comment:
the name doesn't line up with `count_map_map`; however, I prefer this naming to the above. do you have any ideas for a better name for that lemma? maybe `count_map_of_injective`? or `count_eq_count_map`?

Reply:
I think the above should be `count_map_of_injective` and this one is fine.

```suggestion
lemma count_le_count_map [decidable_eq β] (l : list α) (f : α → β) (x : α) :
```